js/main.js

```diff
@@ -24,4 +24,70 @@ function submitForm(event) {
   form.reset();
   mainImage.setAttribute('src', 'images/placeholder-image-square.jpg');
   data.entries.unshift(entry);
+  ul.prepend(createEntry(entry));
 }
+
+function createEntry(entry) {
+  var li = document.createElement('li');
+  li.setAttribute('class', 'row');
+
+  var columnDivHead = document.createElement('div');
+  columnDivHead.setAttribute('class', 'column-half flex justify-center');
+
+  var imgHead = document.createElement('img');
+  imgHead.setAttribute('src', entry.photo);
+  imgHead.setAttribute('class', 'cover image-entry');
+
+  var columnDivMain = document.createElement('div');
+  columnDivMain.setAttribute('class', 'column-half');
+
+  var h2Main = document.createElement('h2');
+  h2Main.setAttribute('class', 'h2-stretch h2-entry');
+  h2Main.textContent = entry.title;
+  var pMain = document.createElement('p');
+  pMain.textContent = entry.notes;
+  pMain.setAttribute('class', 'p-entry');
+
+  li.appendChild(columnDivHead);
+  li.appendChild(columnDivMain);
+  columnDivHead.appendChild(imgHead);
+  columnDivMain.appendChild(h2Main);
+  columnDivMain.appendChild(pMain);
+  noClass.setAttribute('class', 'row no-entries text-center hidden');
+  return li;
+}
+var noClass = document.querySelector('#noClass');
+var ul = document.querySelector('#entriesList');
+
+var savedArray = [];
+var savedEntries = localStorage.getItem('javascript-local-storage');
+
+if (savedEntries !== null) {
+  savedArray = JSON.parse(savedEntries);
+}
```

Comment:
What is the purpose of this? All localStorage-related code should be in the `data.js` file, where the `data` variable is located. And you shouldn't be making your own variables to keep track of the entries, you should make use of the `data.entries` array